src/contexts/ConfigurationContext.tsx

```diff
@@ -431,10 +439,12 @@ export const ConfigContextProvider = ({ children }: Props) => {
           config_summary_default_language as SummaryLanguage,
           "auto"
         ),
+        defaultStyle: config_summary_default_style ?? "default",
         summaryNumResults: config_summary_num_results ?? 7,
         summaryNumSentences: config_summary_num_sentences ?? 3,
         summaryPromptName:
           config_summary_prompt_name ?? "vectara-summary-ext-v1.2.0",
+        summaryStyledPrompt: config_summary_styled_prompt ?? "false",
```

Comment:
We should convert this to a boolean value instead of the string "false", like this:

```js
summaryStyledPrompt: isTrue(config_summary_styled_prompt),
```

Note that this also expects the value to be set to "True" with a capital "T" if users want to enable this option.

Reply:
Done